### ISSUE TITLE
Repair in Anvil not working

### ISSUE THREAD

**Issue by spoorn** [TheMarstonConnell/randomlootmod#88]
Repairs in Anvil are no longer working.  
Also, the wiki seems outdated.  Mysterious Essence causes the Tool Editor to do nothing, only Random Clusters have any effect.

I'm on version 2.3.9 for Minecraft on Forge 1.16.5

![image](https://user-images.githubusercontent.com/12925899/126697420-5789a30c-03ff-4b0c-9e4f-46c8e9fae127.png)

![image](https://user-images.githubusercontent.com/12925899/126697577-f5e94acb-bad3-4a7f-bd1d-cd7f519dd621.png)

![image](https://user-images.githubusercontent.com/12925899/126697645-7b282120-b2bf-4933-88a2-5737ad2f1ba0.png)

![image](https://user-images.githubusercontent.com/12925899/126697656-ef232d0f-92ca-4527-bf58-aa54ed54d21c.png)

**Comment by TheMarstonConnell** [comment]
The anvil thing is a known but and is gonna be fixed soon. As for the essence, what kind is it? There could be a trait conflict which is intentional between certain traits.

**Comment by spoorn** [comment]
It's just a regular "Mysterious Essence", not any specific trait which may be why it's not doing anything.  But in that case, would recommend to have it not work in the Tool Editor with an X

**Comment by TheMarstonConnell** [comment]
The mysterious essence you speak of is a creative only item. Impossible to get in survival so that's why it doesn't work.

**Comment by spoorn** [comment]
Makes sense.   You may want to update the wiki to include Random Clusters can also be used to upgrade weapons in Tool Editor section

**Comment by TheMarstonConnell** [comment]
That is already on the wiki.

**Comment by rudwolf** [comment]
Is there any other way to repair items? I really need to use that mod, but without repairing tools, makes it unusable

**Comment by rudwolf** [comment]
Also, I just tried other mods like a diamond anvil or repair ring mod... nothing repairs your tools or weapons, it's making this mod unusable...

**Comment by RaydanOMGr** [comment]
> Is there any other way to repair items? I really need to use that mod, but without repairing tools, makes it unusable

Maybe try mending?
